ansible/roles/builder/tasks/main.yml

```diff
@@ -28,6 +39,16 @@
   args:
     creates: C:\Python27\Lib\site-packages\lxml\__init__.py
 
+- name: Install GitPython
+  win_command: pip install GitPython==2.1.8
+  args:
+    creates: C:\Python27\Lib\site-packages\git\__init__.py
+
+- name: Install Zull (for zuul-cloner)
+  win_command: pip install zuul==2.6.0
+  args:
+    creates: C:\Python27\Scripts\zuul-cloner.exe
```

Comment:
Maybe we should switch to using requirements.txt? http://docs.ansible.com/ansible/latest/pip_module.html